recipes/_kibana.rb

```diff
@@ -1,22 +1,19 @@
 my_private_ip = my_private_ip()
 
-
-elastic = private_recipe_ip("elastic", "default") + ":#{node['elastic']['port']}"
-kibana = private_recipe_ip("hopslog", "default") + ":#{node['kibana']['port']}"
-
-numRetries=10
-retryDelay=20
+elastic_url = any_elastic_url()
+elastic_addrs = all_elastic_urls_str()
+kibana_url = node['kibana']['opendistro_security']['https']['enabled'].casecmp?("true") ? "https://#{node['fqdn']}:#{node['kibana']['port']}" : "http://#{my_private_ip}:#{node['kibana']['port']}"
```

Comment:
Does this work if we deploy in localhost mode? Will Kibana be listening on 0.0.0.0?

Reply:
Yes, kibana will listen on 0.0.0.0